chanbench.hs

```diff
@@ -63,7 +63,7 @@ runtest n test = do
     1 -> do
       replicateM_ n $ writec c (1 :: Int)
       replicateM_ n $ readc c
-    2 -> do
+    2 -> 
```

Comment:
I actually don't mind the unnecessary `do`, it makes it easier to add more statements later. But I don't feel  strongly about it.